build.gradle

```diff
@@ -178,9 +178,7 @@ subprojects { subproject ->
 		testRuntime "org.apache.logging.log4j:log4j-slf4j-impl:$log4jVersion"
 		testRuntime "org.apache.logging.log4j:log4j-jcl:$log4jVersion"
 
-		testCompile("com.willowtreeapps.assertk:assertk:$assertkVersion") {
-				exclude group: 'org.jetbrains.kotlin', module: 'kotlin-reflect'
-		}
+		testCompile("com.willowtreeapps.assertk:assertk-jvm:$assertkVersion")
```

Comment:
Why is the exclusion no longer needed?

Reply:
According my Gradle report the old version is remapped pretty well:
```
+--- com.willowtreeapps.assertk:assertk-jvm:0.12
|    +--- com.willowtreeapps.assertk:assertk-common:0.12
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.2.50 -> 1.2.61
|    |    \--- com.willowtreeapps.opentest4k:opentest4k-common:1.1.0
|    |         \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.2.41 -> 1.2.61
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.2.50 -> 1.2.61
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.2.61
|    |    \--- org.jetbrains:annotations:13.0
|    +--- org.jetbrains.kotlin:kotlin-reflect:1.2.50 -> 1.2.61
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.2.61 (*)
|    \--- com.willowtreeapps.opentest4k:opentest4k-jvm:1.1.0
|         +--- com.willowtreeapps.opentest4k:opentest4k-common:1.1.0 (*)
|         +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.41 -> 1.2.61
|         |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.2.61 (*)
|         |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.61
|         |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.2.61 (*)
|         \--- org.opentest4j:opentest4j:1.1.0
+--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.61 (*)
```

And there is no any issues during running tests against that latest AssertK.